homeworks/vancsikpeti/7_homework/coingecko_api.py

```diff
@@ -0,0 +1,20 @@
+import requests
+
+class CoinGecko:
+
+    def get_data(limit: int):
```

Comment:
ha ezt egy general method-nak szántad amivel bármilyen adatot le lehet húzni az api-ról, akkor mégjobb lenne ha az url és a paraméterek is átadhatóak lennének ennek a metódusnak.
Így ha egy másik endpoint-ról akarnánk adatokat szerezni, ahol más az url, és más paramétereket kell megadni, szintén meg tudnánk tenni ezzel a metódussal.